bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/AbstractTestFilters.java

```diff
@@ -2038,7 +2038,7 @@ public void testPageFilters() throws IOException {
     PageFilter pageFilter = new PageFilter(20);
     scan.setFilter(pageFilter);
     try (ResultScanner scanner = table.getScanner(scan)) {
-      Assert.assertEquals(20, Iterators.size(scanner.iterator()));
+      Assert.assertEquals(100, Iterators.size(scanner.iterator()));
```

Comment:
I don't think this is right, the pageFilter sets the limit to 20, which should be respected.

Reply:
fixed